src/keywordPreservation.ts

```diff
@@ -15,9 +22,9 @@ export const shouldFieldBePreserved = (
 };
 
 export const getPreservableFieldsFromAssets = (
-  asset: any,
-  address: string,
-  keywordMappings: KeywordMappings
+  asset: object,
+  keywordMappings: KeywordMappings,
+  address = ''
```

Comment:
Re-ordering the arguments, particularly making `keywordMappings` second to be more consistent with the other functions of that file.